### ISSUE TITLE
[BUG]: train_rm.py get lower acc!

### ISSUE THREAD

**Issue by Yutongamber** [hpcaitech/ColossalAI#3550]
### 🐛 Describe the bug

hello,
here is a bug, similar with [issues-3534](https://github.com/hpcaitech/ColossalAI/issues/3534)

use default Anthropic/hh-rlhf dataset,
pretrain_model: bigscience/bloom-1b1
batch_size: 1
max_epochs: 1
max_len: 512
loss_fn: log_sig

loss is random change
acc is much lower than than the one reported in the readme.

using the following cmd:
![image](https://user-images.githubusercontent.com/56161755/231709319-c39ab18d-52d0-46e4-835f-0b617e455859.png)

here is the results:
![image](https://user-images.githubusercontent.com/56161755/231711516-9e0f8e4f-bcbf-4db5-973e-efba05a520a5.png)

Is there any advice? 

Thanks!

**Comment by JThh** [comment]
Has it been trained for one full epoch?

**Comment by Yutongamber** [comment]
Yes, it is trained for one full epoch

**Comment by binmakeswell** [comment]
Hi @Yutongamber Maybe it’s an inappropriate sh command. We have fixed it. Thanks. #3490 
 https://github.com/hpcaitech/ColossalAI/blob/main/applications/Chat/examples/train_rm.sh
